### DISCUSSION TITLE
Missing README Bytecode/Opcode Links (NFT Section)

### DISCUSSION THREAD

**Discussion by NemboKid** [Cyfrin/foundry-full-course-cu#1090] [General]
Hello!

I noticed that the links that are shown here in the video [https://youtu.be/sas02qSFZ74?t=37720](https://youtu.be/sas02qSFZ74?t=37720) are missing in the actual README. Any chance to add them for those who wants to dig deeper? Thanks!

**Comment by 0xusmanf** [comment]
You can find them [here](https://github.com/smartcontractkit/full-blockchain-solidity-course-js#abiencode--abiencodepacked).
